test/metabase/query_processor/middleware/wrap_value_literals_test.clj

```diff
@@ -211,6 +211,13 @@
               {:source-query {:source-table $$checkins
                               :filter       [:> $date "2014-01-01"]}}))))))
 
+(deftest ^:parallel wrap-literals-around-expressions
+  (testing "does wrapping literals work against expressions? (#27185)"
+    (wrap-value-literals
```

Comment:
I assume here we are testing that `qp.wrap-value-literals/wrap-value-literals` does not throw. If my assumption is correct, being explicit about that would make the code easier to reason about.

Reply:
Thanks, dumb miss on my end, repl testing and forgot to make an assertion.